<!DOCTYPE HTML>
<html>
	<head>
		<title>Phillips Lab</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />		
		<link rel="icon" href="images/brain-solid.svg">
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<div class="logo">
							<span class="fas fa-brain"></span>
						</div>
						<div class="content">
							<div class="inner">
								<h1>Phillips Lab</h1>
								<p>A Computational Biology and Cognitive Neuroscience lab</p>
							</div>
						</div>
						<nav>
							<ul>
								<li><a href="#projects">Projects</a></li>
								<li><a href="#works">Works</a></li>
								<li><a href="#people">People</a></li>
								<li><a href="#about">About</a></li>
							</ul>
						</nav>
					</header>

				<!-- Main -->
					<div id="main">

						<!-- Projects -->
							<article id="projects">
								<h2 class="major">Projects</h2>
								<!-- <span class="image main"><img src="images/projects_img.jpg" alt="" /></span> -->
								<h3 class="minor">Environmental Effects on Protein-Protein Interactions</h3>
								<p>A complete understanding of how environmental factors influence HIV transmission is needed for the development of an effective HIV vaccine. I develop and utilize state-of-the-art modelling and simulation methods to investigate protein-protein interactions involved in viral transmission at the molecular level, and determine how relevant environmental factors (such as salt concentration and pH) affect these interactions. This project is a collaboration with researchers at Los Alamos National Laboratory.</p>
								<h3 class="minor">Mutational Analysis of Metalloproteins</h3>
								<p>Modelling and simulation provide a rational approach for predicting how mutations to metalloprotein enzymes will degrade or enhance catalysis of viable substrates. These predictions greatly aid the search for useful mutations by prioritizing the screening order of mutants in the laboratory, lowering costs and producton times. Key targets have been enzymes which degrade chemical warfare nerve agents and may be used for therapeutic or sanitation purposes. This project is a collaboration with researchers at Los Alamos National Laboratory.</p>		
								<h3 class="minor">Multiscale Modeling and Simulation of Multidrug Resistance</h3>					
								<p>Bacteria have developed several mechanisms which contribute to multidrug resistance and hinder the development of new antibiotic treatments. If the mechanisms involved in resistance were more fully understood, novel treatments might be developed which could rescue our current stream of antibiotics. I use modelling and simulation to study the structure and function of multidrug resistance efflux pumps, one of the main contributors to antibiotic resistance. I aim to uncover a complete mechanistic understanding of pump function in order to aid efforts to effectively subvert their function. This project is a collaboration with researchers at Los Alamos National Laboratory.</p>	
								<h3 class="minor">Molecular Simulation Analysis</h3>	
								<p>There are many biological and chemical processes which can be studied in remarkable detail using computational modeling and simulation. The significant computational cost of these approaches suggests that rigorous analysis of the resulting data is necessary to justify the consumed computational resources, and state-of-the-art statistical and machine learning methods are poised to fill this need. However, these methods are often developed outside of the scientific domains where they are applied, and the nuances faced when working with real data make it difficult to discern when these approaches are achieving their intended purpose. I use domain knowledge to construct model-based validation frameworks which help to resolve such issues. Past domains of interest include intrinsically disordered and natively folded proteins. This project is a collaboration with researchers at the University of California, Merced.</p>
								<h3 class="minor">Robot Prefrontal Cortex (PFC) Working Memory Toolkit (WMtk)</h3>
								<p>One past project focused on the development of biologically inspired computational mechanisms for effective robot learning and control. In particular, David Noelle (Univ. of Calif., Merced), and I developed a software toolkit that allows for the easy integration of a powerful computational neuroscience model of working memory into robotic systems. Current work involves combining the toolkit with models of other brain systems and creating more efficient knowledge representation structures which are more flexible and comprehensive than those currently used by toolkit. This model of working memory has been used to train robots to perform standard laboratory tests of working memory function, such as the delayed saccade task, as well tasks in robot navigation, motor skill learning, and object manipulation.</p>
							</article>

						<!-- Works -->
							<article id="works">
								<h2 class="major">Work</h2>
								<!-- <span class="image main"><img src="images/papers_img.jpg" alt="" /></span> -->

								<div class="container" id="content">
									<h3 class="minor">Papers</h3>
									<ul>
									  <li>Khan, N. and Phillips, J. L. (in press). Combined model for sensory-based and feedback-based task switching: solving hierarchical reinforcement learning problems statically and dynamically with transfer learning. In <em>Proceedings of the 32nd International Conference on Tools with Artificial Intelligence</em>.</li>
									  <li>Syzonenko, I. and Phillips, J. L. (2020). Accelerated protein folding using greedy-proximal A*. <em>Journal of
Chemical Theory and Computation</em>. <a title="PDF" href="https://doi.org/10.1021/acs.jcim.9b01194">[LINK]</a></li>
									  <li>Williams, A. S. and <b>Phillips, J. L.</b> (2020). Transfer reinforcement learning using output-gated working memory. In <em>Proceedings of the 34th AAAI Conference on Artificial Intelligence</em>, New York, NY. <a title="PDF" href="https://aaai.org/ojs/index.php/AAAI/article/view/5488/5344">[PDF]</a> or <a title="PDF" href="papers/WilliamsPhillips-AAAI-2020-preprint.pdf">[preprint-PDF]</a></li>
									  <li>Connors, K. A., Beasley, A., Barron, M. G., Belanger, S. E., Bonnell, M., Brill, J. L., de Awart, D., Kienzler, A., Krailler, J., Otter, R., <b>Phillips, J. L.</b>, and Embry, M. R. (2019). Creation of a curated aquatic toxicology database: EnviroTox. <em>Environmental Toxicology and Chemistry</em>. <a title="PDF" href="https://doi.org/10.1002/etc.4382">[LINK]</a></li>
									  <li>Morton, S. P., Phillips, J. B., and <b>Phillips, J. L.</b> (2019). The molecular basis of pH-modulated HIV gp120 binding revealed. <em>Evolutionary Bioinformatics</em>. <a title="PDF" href="https://doi.org/10.1177/1176934319831308">[LINK]</a></li>
									  <li>Morton, S. P., Howton, J., and <b>Phillips, J. L.</b> (2018). Sub-class differences of pH-dependent HIV GP120-CD4 interactions. In <em>Proceedings of the 9th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics (CSBW)</em>, Washington, DC. <a title="PDF" href="https://doi.org/10.1145/3233547.3233711">[PDF]</a> or <a title="PDF" href="papers/Morton-etal-ACMBCB-2018.pdf">[PDF]</a></li>
									  <li><b>Phillips, J. L.</b>, Colvin, M. E., and Newsam, S. (2018). Dimensionality estimation of protein dynamics using polymer models. In <em>Proceedings of the 9th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics (CSBW)</em>, Washington, DC. <a title="PDF" href="https://doi.org/10.1145/3233547.3233713">[PDF]</a> or <a title="PDF" href="papers/PhillipsColvinNewsam-ACMBCB-2018.pdf">[PDF]</a></li>
									  <li>Syzonenko, I. and <b>Phillips, J. L.</b> (2018). Hybrid spectral/subspace clustering of molecular dynamics simulations. In <em>Proceedings of the 9th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics</em>, Washington, DC. <a title="PDF" href="https://doi.org/10.1145/3233547.3233595">[PDF]</a> or <a title="PDF" href="papers/SyzonenkoPhillips-ACMBCB-2018.pdf">[PDF]</a></li>
									  <li>Jovanovich, M. P. and <b>Phillips, J. L.</b> (2018). N-task learning: solving multiple or unknown numbers of reinforcement learning problems. In <em>Proceedings of the 40th Annual Meeting of the Cognitive Science Society</em>, Madison, WI. <a title="PDF" href="https://mindmodeling.org/cogsci2018/papers/0126/0126.pdf">[PDF]</a> <a title="HTML" href="https://mindmodeling.org/cogsci2018/papers/0126/index.html">[HTML]</a></li>
									  <li>Williams, A. S. and <b>Phillips, J. L.</b> (2018). Multilayer context reasoning in a neurobiologically inspired working memory model for cognitive robots. In <em>Proceedings of the 40th Annual Meeting of the Cognitive Science Society</em>, Madison, WI. <a title="PDF" href="https://mindmodeling.org/cogsci2018/papers/0510/0510.pdf">[PDF]</a> <a title="HTML" href="https://mindmodeling.org/cogsci2018/papers/0510/index.html">[HTML]</a></li>
									  <li>Mueller, R. T., Travers, T., Cha H.-J., <b>Phillips, J.</b>, Gnanakaran, S., and Pos, K. M. (2017). Switch loop flexibility affects substrate transport of the AcrB efflux pump. <em>Journal of Molecular Biology</em>, 429 (24), 3863-3874. <a href="https://doi.org/10.1016/j.jmb.2017.09.018">[LINK]</a></li>
									  <li>Morton, S. P., Phillips, J. B., and <b>Phillips, J. L.</b> (2017). High-throughput structural modeling of the HIV transmission bottleneck. <em>2017 IEEE International Conference on Bioinformatics and Biomedicine Workshops</em>, Kansas City, MO. <a href="http://ieeexplore.ieee.org/document/8217952/">[PDF]</a> or <a href="papers/Morton-etal-BIBM-2017-preprint.pdf">[preprint-PDF]</a></li>
									  <li>Howton, J. and <b>Phillips, J. L.</b> (2017). Computational modeling of pH-dependent gp120-CD4 interactions in founder and chronic HIV strains. <em>Proceedings of the 8th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics (CSBW)</em>, Boston, MA. <a href="http://dl.acm.org/ft_gateway.cfm?id=3107506&amp;ftid=1899882&amp;dwn=1&amp;CFID=811759641&amp;CFTOKEN=79968238">[PDF]</a> or <a href="papers/HowtonPhillips-ACMBCB-CSBW-2017.pdf">[PDF]</a></li>
									  <li>DuBois, G. M. and <b>Phillips, J. L.</b> (2017). Working memory concept encoding using holographic reduced representations. <em>Proceedings of the 28th Modern Artificial Intelligence and Cognitive Science Conference</em>, Fort Wayne, IN. <a href="papers/DuBoisPhillips-MAICS-2017.pdf">[PDF]</a></li>
									  <li><b>Phillips, J. L.</b>, and Gnanakaran, S. (2015). A data-driven approach to modeling the tripartite structure of multidrug resistance efflux pumps. <em>Proteins: Structure, Function, and Bioinformatics</em>, 83 (1), 46-65. <a href="http://onlinelibrary.wiley.com/doi/10.1002/prot.24632/pdf">[PDF]</a></li>
									  <li>Gottardo, R., Bailer, R. T., Korber, B T., Gnanakaran, S., <b>Phillips, J.</b>, Shen, X., Tomaras, G. D., Turk, E., Imholte, G., Eckler, L., Wenschuh, H., Zerweck, J., Greene, K., Gao, H., Berman, P. W., Francis, D., Sinangil, F., Lee, C., Nitayaphan, S., Rerks-Ngarm, S., Kaewkungwal, J., Pitisuttithum, P., Tartaglia, J., Robb, M. L., Michael, N. L., Kim, J. H., Zolla-Pazner, S., Haynes, B. F., Mascola, J. R., Self, S., Gilbert, P., Montefiori, D. C. (2013). Plasma IgG to linear epitopes in the V2 and V3 regions of HIV-1 gp120 correlate with a reduced risk of infection in the RV144 vaccine efficacy trial. <em>PLoS One</em>. <a href="http://www.plosone.org/article/fetchObject.action;jsessionid=0A14549F2116999F71EC41C9911D7115?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0075665&amp;representation=PDF">[PDF]</a></li>
									  <li>Stieh, D., <b>Phillips J. L.</b>, Rogers P. M., King, D. F., Cianci, G. C., Jeffs, S. A., Gnanakaran, S., and Shattock, R. J. (2013). Dynamic electrophoretic fingerprinting of the HIV-1 envelope glycoprotein. <em>Retrovirology, </em>10 (33). <a title="PDF" href="http://www.retrovirology.com/content/pdf/1742-4690-10-33.pdf">[PDF]</a></li>
									  <li><b>Phillips, J. L.</b>, Colvin, M. E., and Newsam, S. (2011). Validating clustering of molecular dynamics simulations using polymer models. <em>BMC Bioinformatics</em>, 12 (1), 445. <a title="PDF" href="http://www.biomedcentral.com/content/pdf/1471-2105-12-445.pdf">[PDF]</a></li>
									  <li>Yamada, J., <b>Phillips, J. L.</b>, Patel, S., Goldfien, G., Calestagne-Morelli, A., Huang, H., Reza, R., Acheson, J., Krishnan, V. V., Newsam, S., Gopinathan, A., Lau, E. Y., Colvin, M. E., Uversky V. N., and Rexach M. F. (2010). A bimodal distribution of two distinct categories of instrinsically-disordered structures with separate functions in FG nucleoporins. <em>Molecular and Cellular Proteomics</em>, 9, 2205-2224. <a title="PDF" href="http://www.mcponline.org/content/early/2010/04/05/mcp.M000035-MCP201.full.pdf">[PDF]</a></li>
									  <li>Lau, E. Y., <b>Phillips, J. L.</b>, and Colvin, M. E. (2009). Molecular dynamics simulations of highly charged green fluorescent proteins. <em>Molecular Physics: An International Journal at the Interface Between Chemistry and Physics</em>, 107 (8), 1233-1241. <a title="PDF" href="http://www.tandfonline.com/doi/pdf/10.1080/00268970902845305">[PDF]</a></li>
									  <li><b>Phillips, J. L.</b>, Colvin, M. E., Lau, E. Y., and Newsam, S. (2008). Analyzing dynamical simulations of intrinsically disordered proteins using spectral clustering. In <em>IEEE BIBM 2008 Workshop on Computational Structural Bioinformatics</em>, Philadelphia, PA. <a title="PDF" href="http://ieeexplore.ieee.org/ielx5/4675626/4686198/04686204.pdf">[PDF]</a></li>
									  <li>Tugcu, M., Wang, X., Hunter, J. E., <b>Phillips, J.</b>, Noelle, D., and Wilkes, D. M. (2007). A computational neuroscience model of working memory with application to robot perceptual learning. In <em>Proceedings of the 3rd International Conference on Computational Intelligence</em>, Banff, Alberta, Canada. <a title="PDF" href="http://citeseerx.ist.psu.edu/viewdoc/download;jsessionid=DBDA7425C118255B69B4A7BEB56CFBC5?doi=10.1.1.119.3407&amp;rep=rep1&amp;type=pdf">[PDF]</a></li>
									  <li><b>Phillips, J. L.</b>, and Noelle, D. C. (2006). Working memory for robots: inspirations from computational neuroscience. In <em>Proceedings of the 5th International Conference on Development and Learning</em>, Bloomington, IN. <a title="PDF" href="http://www.mentaldev.org/icdl06/papers/93_Phillips_paper.pdf">[PDF]</a></li>
									  <li><b>Phillips, J. L.</b>, and Noelle, D. C. (2005). A biologically inspired working memory framework for robots. In <em>Proceedings of the 14th IEEE International Workshop on Robot and Human Interactive Communication</em>, Nashville, TN. <a title="PDF" href="http://ieeexplore.ieee.org/xpls/abs_all.jsp?arnumber=1513845">[PDF]</a></li>
									  <li><b>Phillips, J. L.</b>, and Noelle, D. C. (2005). A biologically inspired working memory framework for robots. In <em>Proceedings of the 27th Annual Meeting of the Cognitive Science Society</em>, Stresa, Italy.</li>
									  <li><b>Phillips, J. L.</b>, Kogekar, S., and Adams, J. A. (2004). Emergency automated response system (EARS). In <em>Proceedings of the 48 Annual Meeting of the Human Factors and Ergonomics Society</em>, New Orleans, LA. <a title="PDF" href="http://pro.sagepub.com/content/48/5/851">[PDF]</a></li>
									  <li><b>Phillips, J. L.</b>, and Noelle, D. C. (2004). Reinforcement learning of dimensional attention for categorization. In <em>Proceedings of the 26th Annual Meeting of the Cognitive Science Society</em>, Chicago, IL. <a title="PDF" href="http://130.203.133.150/viewdoc/summary?doi=10.1.1.122.7278">[PDF]</a></li>
									</ul>
									<h3 class="minor">Ph.D. Dissertation</h3>
									<ul>
									  <li>Phillips, J. L. (2012). <em>Validation of computational approaches for studying disordered and unfolded protein dynamics using polymer models</em>. University of California, Merced. <a href="https://www.cs.mtsu.edu/~jphillips/papers/Dissertation-JLP-20120717.pdf">[PDF]</a></li>
									</ul>
									<h3 class="minor">M.S. Thesis</h3>
									<ul>
									  <li>Phillips, J. L. (2004). <em>Reinforcement learning of dimensional attention for categorization</em>. Vanderbilt University. <a href="https://www.cs.mtsu.edu/~jphillips/papers/Thesis-JLP-20040401.pdf">[PDF]</a></li>
									</ul>
									<h3 class="minor">Posters</h3> 
									<ul>
										<li><b>Phillips, J. L.</b>, Lau, E. Y., Colvin, M. E., and Newsam, S. (2010). Dimensionality reduction reveals differences between disordered protein dynamics and early-stage protein folding dynamics. <em>24th Annual Symposium of the Protein Society</em>, San Diego, CA.</li>
										<li><b>Phillips, J. L.</b>, Lau E. Y., Rexach, M., Newsam, S., and Colvin, M. E. (2010). Differences between unfolded and disordered protein dynamics. <em>1st Gordon Research Conference on Intrinsically Disordered Proteins</em>, Davidson, SC.</li>
									    <li><b>Phillips, J. L.</b>, Lau E. Y., Krishnan, V. V., Rexach, M., Newsam, S., and Colvin, M. E. (2010). Metric scaling for dimensionality reduction of disordered protein dynamics. <em>54th Annual Meeting of the Biophysical Society</em>, San Francisco, CA. (2010 Student Research Achievement Award)</li>
									    <li><b>Phillips, J. L.</b>, Manilay, J. O., and Colvin, M. E. (2010). Analytic parameter fitting in stochastic stem cell models. <em>54th Annual Meeting of the Biophysical Society</em>, San Francisco, CA.</li>
									    <li><b>Phillips, J. L.</b>, Lau E. Y., Krishnan, V. V., Rexach, M., Newsam, S., and Colvin, M. E. (2009). Dynamics analysis of unstructured FG-nucleoporins. <em>23nd Annual Symposium of the Protein Society</em>, Boston, MA. (2009 Best Student Poster Award)</li>
										<li><b>Phillips, J. L.</b>, Lau E. Y., Krishnan, V. V., Rexach, M., Newsam, S., and Colvin, M. E. (2008). Characterizing intrinsically disordered FG-nucleoporins using molecular dynamics. <em>22nd Annual Symposium of the Protein Society</em>, San Diego, CA.</li>
									</ul>
							</div>
							</article>

						<!-- People -->
							<article id="people">
								<h2 class="major">People</h2>
								<!-- <span class="image main"><img src="images/people_img.jpg" alt="" /></span> -->
								<div class="container" id="content">
	
									<h2>Current Students</h2>
														
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://cs.mtsu.edu/~jphillips/images/3604672.jpg" alt="[Image (JPEG 23K): Scott Morton]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										  <h2>Scott P. Morton</h2>
										  <b>Graduate Student</b><br>
										  Ph.D. Program - Computational Science<br>
										  Email: spm3c[at]mtmail.mtsu.edu<br>
										  WWW: <a href="//www.cs.mtsu.edu/~spm3c/">http://www.cs.mtsu.edu/~spm3c/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Currently researching the ElectroStatic Surface Charge Pipeline for HIV antibody binding characteristics. Additionally converting all Bash scripts into a unified python language format with a json based configuration file. Further research goals include a rewrite of frodaN for ease of use and parallel processing potentials involving CUDA and/or MPI, Massively Parallel Processing (MPP) of model protein sequences for predicting binding characteristics of HIV and HIV antibodies, potential electronic circuitry characteristics of protein sequences as a means of binding energy.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01102771.JPG" alt="[Image (JPEG 100K): Arthur Williams]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										  <h2>Arthur S. Williams</h2>
										  <b>Graduate Student</b><br>
										  Ph.D. Program - Computational Science<br>
										  Email: asw3x[at]mtmail.mtsu.edu<br>
										  WWW: <a href="//www.cs.mtsu.edu/~asw3x/">http://www.cs.mtsu.edu/~asw3x/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Hierarchical and indirection-based working memory models for task generalization in partially observable reinforcement learning domains.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01357523.JPG" alt="[Image (JPEG 100K): Will Haase]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Will H. Haase</h2>
										<b>Undergraduate Student</b><br>
										B.S. Program - Computer Science<br>
										Email: whh2p[at]mtmail.mtsu.edu<br>
										WWW: None<br>
									  </div>
									  <div class="col-sm-6">
										<p>Minimizing catatrophic interference via unitization and generative recurrent neural network models.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01402667.JPG" alt="[Image (JPEG 100K): Nibraas Khan]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Nibraas A. Khan</h2>
										<b>Undergraduate Student</b><br>
										B.S. Program - Computer Science<br>
										Email: nak2z[at]mtmail.mtsu.edu<br>
										WWW: None<br>
									  </div>
									  <div class="col-sm-6">
										<p>Combined working memory and N-task models for partially-, non-observable reinforcement learning problems.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01365683.JPG" alt="[Image (JPEG 42K): Lucas Remedios]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Lucas Remedios</h2>
										<b>Undergraduate Student</b><br>
										B.S. Program - Computer Science<br>
										Email: lwr2k[at]mtmail.mtsu.edu<br>
										WWW: <a href="https://www.cs.mtsu.edu/~lwr2k/">https://www.cs.mtsu.edu/~lwr2k/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Keras/TensorFlow tools for N-task learning.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01371449.JPG" alt="[Image (JPEG 100K): Matt Radice]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Matthew T. Radice</h2>
										<b>Graduate Student</b><br>
										Ph.D. Program - Computational Science<br>
										Email: mtr3t[at]mtmail.mtsu.edu<br>
										WWW: None<br>
									  </div>
									  <div class="col-sm-6">
										<p>Limitations of Q-Learning for partially observable reinforcement learning domains and compensatory mechanisms via working memory.</p>
									  </div>
									</div>
								
									<hr>
									<h2>Past Students</h2>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01218003.JPG" alt="[Image (JPEG 100K): Ivan Syzonenko]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Ivan Syzonenko</h2>
										<b>Graduate Student</b><br>
										Ph.D. Program - Computational Science<br>
										Email: is2k[at]mtmail.mtsu.edu<br>
										WWW: <a href="//www.cs.mtsu.edu/~is2k/">http://www.cs.mtsu.edu/~is2k/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Ivan's research has focused on understanding and overcoming the limitations of machine learning algorithms, particularly clustering, in biomolecular simulation analysis. More recently, he is developing and testing methods for accelerated unbiased targeted MD simulations using classical pathfinding approaches with a focus on protein folding.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01374152.JPG" alt="[Image (JPEG 100K): Huizhi Wang]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Huizhi Wang</h2>
										<b>Graduate Student</b><br>
										M.S. Program - Computer Science<br>
										Email: hw3m[at]mtmail.mtsu.edu<br>
										WWW: <a href="//www.cs.mtsu.edu/~hw3m/">http://www.cs.mtsu.edu/~hw3m/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Dimensional attention mechanisms for holographic reduced representational encodings and category learning.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01334416.JPG" alt="[Image (JPEG 100K): Ngozi Omatu]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Ngozi C. Omatu</h2>
										<b>Undergraduate Student</b><br>
										B.S. Program - Biology<br>
										Email: nco2f[at]mtmail.mtsu.edu<br>
										WWW: None<br>
									  </div>
									  <div class="col-sm-6">
										<p>Dimensional attention for working memory: accelerated early learning with asymptotically optimal performance.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										  <img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01276101.JPG" alt="[Image (JPEG 100K): Mike Jovanovich]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Mike Jovanovich</h2>
										<b>Graduate Student</b><br>
										M.S. Program - Computer Science<br>
										Graduation: Fall 2017<br>
										Email: mpj2n[at]mtmail.mtsu.edu<br>
										WWW: <a href="//www.cs.mtsu.edu/~mpj2n/">http://www.cs.mtsu.edu/~mpj2n/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Neurobiologically plausible models of working memory, task switching, and task generalization.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01238151.JPG" alt="[Image (JPEG 100K): Joshua Arnold]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Joshua M. Arnold</h2>
										<b>Undergraduate Student</b><br>
										B.S. Program - Computer Science<br>
										Email: jma5x[at]mtmail.mtsu.edu<br>
										WWW: <a href="//www.cs.mtsu.edu/~jma5x/">http://www.cs.mtsu.edu/~jma5x/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Coupled action-working memory learning for partially observable reinforcement learning problems.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://cs.mtsu.edu/~jphillips/images/SAM_2875-1.jpg" alt="[Image (JPEG 16K): Cody Crawford]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Cody Crawford</h2>
										<b>Graduate Student</b><br>
										M.S. Program - Computer Science<br>
										Graduation: Summer 2017<br>
										Email: crn2k[at]mtmail.mtsu.edu<br>
										WWW: <a href="//www.cs.mtsu.edu/~crn2k/">http://www.cs.mtsu.edu/~crn2k/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Journalism today is dealing with so much data that better methods are needed to process it.  Latent Dirichlet Allocation (LDA) is often used to sort text into topics.  The Afghan War Diary (AWD) was processed with LDA and model trees to ascertain fatality numbers.  The AWD was used in a separate study that analyzed the documents with point process modeling (PPM) to predict where conflicts would occur in space and time.  We have combined the two approaches in this study, hopeful that the results will allow us to predict where and when conflicts occur and if the fatality numbers can also be obtained in reference to that.  We anticipate that our results will show that PPM combined with LDA and model trees will give more useful results than using either of the methods separately.</p>
									  </div>
									</div>
									
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01312796.JPG" alt="[Image (JPEG 100K): Jonathan Howton]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Jonathan Howton</h2>
										<b>Graduate Student</b><br>
										M.S. Program - Computer Science<br>
										Graduation: Spring 2017<br>
										Email: jh6w[at]mtmail.mtsu.edu<br>
										WWW: <a href="//www.cs.mtsu.edu/~jh6w/">http://www.cs.mtsu.edu/~jh6w/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Jonathan's research focuses on using high-throughput structural analysis to understand how environmental factors affect protein binding particularly with regard to viral transmission.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://cs.mtsu.edu/~jphillips/images/Grayson_100x100px.jpg" alt="[Image (JPEG 7K): Grayson Dubois]" width="200" height="200">
										<img class="w-75" src="https://cs.mtsu.edu/~jphillips/images/WMtk_Architecture-BOTH.png" alt="[Image (JPEG 178K): WMtk]" width="200" height="100">
									  </div>
									  <div class="col-sm-4">
										<h2>Grayson Dubois</h2>
										<b>Undergraduate Student</b><br>
										B.S. Program - Computer Science - B.S. Spring 2017<br>
										Email: Grayson.Dubois[at]mtsu.edu<br>
										WWW: <a href="//www.cs.mtsu.edu/~gmd2n/">http://www.cs.mtsu.edu/~gmd2n/</a><br>
									  </div>
									  <div class="col-sm-6">
										  <p>Grayson's research looks into a new method of representing concepts in artificial neural networks (ANNs) that mimic human working memory systems. This new method uses something called Holographic Reduced Representations (HRRs), which are powerful tools capable of representing compositional structure in distributed representations. Grayson has developed an engine for encoding and decoding HRRs and is working on integrating it into the Working Memory toolkit, a software library written in ANSI C++ designed to allow researchers to write simulations of learning tasks using ANNs modeled after working memory. The current toolkit requires the programmer to explicitly provide methods to convert concepts used by working memory from symbolic encodings (SE) to distributed encodings (DE). Grayson's HRR Engine will automate the process of SE/DE conversion, thus taking the burden off of the programmer and opening the door to future possibilities not possible with previous DE methods. These include but are not limited to the chunking of similar concepts in memory, transferability of learned behaviors between tasks, and long term memory.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01168478.JPG" alt="[Image (JPEG 100K): Gary Hammock]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Gary Hammock</h2>
										<b>Graduate Student</b><br>
										M.S. Program - Computer Science<br>
										Graduation: Fall 2016<br>
										Email: glh2y[at]mtmail.mtsu.edu<br>
										WWW: <a href="//www.cs.mtsu.edu/~glh2y/">http://www.cs.mtsu.edu/~glh2y/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Gary's research focuses on the development and testing of novel simulation-inspired methods for cryptography.</p>
										</div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01299719.JPG" alt="[Image (JPEG 100K): Robert Myers]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Robert Myers</h2>
										<b>Graduate Student</b><br>
										M.S. Program - Computer Science<br>
										Graduation: Spring 2016<br>
										Email: rvm2d[at]mtmail.mtsu.edu<br>
										WWW: <a href="//www.cs.mtsu.edu/~rvm2d/">http://www.cs.mtsu.edu/~rvm2d/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Robert's research focused on the development and testing of distributed pathfinding algorithms.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://cs.mtsu.edu/~jphillips/images/murphy_thumbnail.png" alt="[Image (PNG 20K): Michael Murphy]" width="200" height="200">
										<img class="w-75" src="https://cs.mtsu.edu/~jphillips/images/fractal.png" alt="[Image (PNG 95K): Fractal]" width="200" height="100">
									  </div>
									  <div class="col-sm-4">
										  <h2>Michael Murphy</h2>
										  <b>Graduate Student</b><br>
										  M.S. Program - Computer Science<br>
										  Graduation: Spring 2016<br>
										  Email: mcm7f[at]mtmail.mtsu.edu<br>
										  WWW: <a href="//www.cs.mtsu.edu/~mcm7f/">http://www.cs.mtsu.edu/~mcm7f/</a><br>
									  </div>
									  <div class="col-sm-6">
										<p>Fractal dimension is a number that describes the self-similarity, or "complexity", of a geometry.  In image processing, fractal dimension is often used as a novel method for contrasting and comparing image content.  The Box-Counting Algorithm is one of the most popular methods of computing an estimate for the fractal dimension of an image, but the algorithm is influenced by many factors such as filtering and noise.  Our research found a relationship between dimensional estimations and the variability in those estimations when using the Box-Counting Algorithm in the presence of increasing levels of uniform noise. This relationship provides a way to strengthen relative dimensional rankings between noisy images.</p>
									  </div>
									</div>
								
									<hr>
									<div class="row">
									  <div class="col-sm-2 text-center">
										<img class="w-75" src="https://ssb.mtsu.edu/mtimages/M01186927.JPG" alt="[Image (JPEG 100K): Stephen Kinser]" width="200" height="200">
									  </div>
									  <div class="col-sm-4">
										<h2>Stephen Kinser</h2>
										<b>Undergraduate Student</b><br>
										B.S. Program - Computer Science<br>
										Email: sdk2v[at]mtmail.mtsu.edu<br>
										WWW: <a href="//www.cs.mtsu.edu/~sdv2k/">http://www.cs.mtsu.edu/~sdv2k/</a><br>
									  </div>
									  <div class="col-sm-6">
										  <p>Stephen's research focused on converting a semi-autotmated protein electrostatics pipeline to a fully-automated version which requires significantly less user intervention.</p>
									  </div>
									</div>
								
									  </div>
							</article>

						<!-- About -->
							<article id="about">
								<h2 class="major">About</h2>
								<div class="row">
									<div class="col-sm-2">
									  <img class="img-fluid rounded" src="https://cs.mtsu.edu/~jphillips/images/jlp-3.jpg" alt="[Image (JPEG 27K): Joshua L. Phillips]">
									</div>
									<div class="col-sm-4">
									   <h2>Joshua L. Phillips</h2>
									   Associate Professor<br>
									   Department of Computer Science<br>
									   College of Basic and Applied Sciences<br>
									   Middle Tennessee State University<br>
									  <!-- <img class="img-fluid float-right w-25" alt="[Image (PNG 2K): QR for Homepage]" src="https://cs.mtsu.edu/~jphillips/images/QR.png" width="100" height="100"> -->
									   <a href="https://cs.mtsu.edu/~jphillips/PhillipsJL-CV.pdf">Curriculum Vitae</a>
									  <br>
									  <br>
									  <h3>Research Interests</h3>
									  <p>My basic research interests are in computational biophysics and cognitive science. I focus primarily on the development of novel computational methods for addressing existing scientific or engineering problems related to molecular or structural biology. My background is in machine learning and neural networks, so my work often employs or is inspired by algorithmic approaches from these fields.</p>
									</div>
									<div class="col-sm-6">
									  <h3>Contact</h3>
									  Office: KOM 356<br>
									  1301 East Main Street<br>
									  MTSU Box 48<br>
									  Mufreesboro, TN 37132<br>
									  615-494-7965 (office)<br>
									  615-898-2397 (department)<br>
									  615-898-5567 (fax)<br>
									  Email: <a href="mailto:Joshua.Phillips@mtsu.edu">Joshua.Phillips@mtsu.edu</a><br>
									  WWW: <a href="http://www.cs.mtsu.edu/~jphillips/">http://www.cs.mtsu.edu/~jphillips/</a><br>
									  GITHUB: <a href="http://github.com/jlphillipsphd/">http://github.com/jlphillipsphd/</a><br>
									  SLACK: <a href="https://phillipslab.slack.com/">https://phillipslab.slack.com/</a><br>
									</div>
									  </div>
							</article>
					</div>

				<!-- Footer -->
					<footer id="footer">
						<p class="copyright">&copy; Phillips Lab </a></p>
					</footer>

			</div>

		<!-- BG -->
			<div id="bg"></div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
